pkg/iter/sc_finder.go

```diff
@@ -80,7 +80,7 @@ func (m *SubclusterFinder) FindStatefulSets(ctx context.Context, flags FindFlags
 // FindServices returns service objects that are in use for subclusters
 func (m *SubclusterFinder) FindServices(ctx context.Context, flags FindFlags) (*corev1.ServiceList, error) {
 	svcs := &corev1.ServiceList{}
-	if err := m.buildObjList(ctx, svcs, flags); err != nil {
+	if err := m.buildObjList(ctx, svcs, flags, vapi.MainCluster); err != nil {
```

Comment:
This is correct, just because we know how FindServices is called, but to be consistent with the other exported APIs, we should take the sandbox in as input parm to FindServices. That will keep this package general purpose.